docs/Release_notes.md

```diff
@@ -1,5 +1,13 @@
 V2X-Hub Release Notes
 ----------------------------
+Version 7.2.2, released May 9th, 2022
+--------------------------------------------------------
+**Summary:**
+V2X Hub release 7.2.2 includes added Hotfix for CARMACloud Plugin to prevent repeated TCM broadcast.V2xhub can control the number of times each TCM being repeatedly broadcast upon receiving TCMs from carma-cloud. After it repeatedly broadcasts the TCMs, it waits for a configured time duration before it sends a time out message to carma-cloud notifying that no response from CMV for TCMs. The no response message will be displayed as a warning on the v2xhub admin user interface. If V2xHub receives an acknowledgement from CMV, it will stop broadcasting TCMs and display the acknowledgement on the v2xhub admin user interface:
```

Comment:
prevent? or allow?

Reply:
Replace with the following:
V2X Hub release 7.2.2 includes added Hotfix for CARMACloud Plugin to configure repeated TCM broadcast. V2xhub can control the number of times each TCM being repeatedly broadcast upon receiving TCMs from carma-cloud until an TCM acknowledgement is received from the vehicle. After waiting for a configurable time duration to receive TCM acknowledgement, it sends a time out message to carma-cloud notifying that no response from CMV for TCMs. The no response message will be displayed as a warning on the v2xhub admin user interface.  If V2xHub receives an acknowledgement from CMV, it will stop broadcasting TCMs and display the acknowledgement on the v2xhub admin user interface and sends the acknowledgement to carma-cloud: